그리디/구명보트/PGS_구명보트_공진호.java

```diff
@@ -0,0 +1,32 @@
+import java.util.Arrays;
+
+public class PGS_구명보트_공진호 {
+    public int solution(int[] people, int limit) {
+        Arrays.sort(people);
+        int p = 0;
+        int q = people.length - 1;
+
+        int cnt = 0;
+        while(p<=q){
+            int lightOne = people[p];
+            int heavyOne = people[q];
+            if(lightOne + heavyOne <= limit){
+                p++;
+                q--;
+            }else{
+                q--;
+            }
```

Comment:
```suggestion
            if(lightOne + heavyOne <= limit){
                p++;
            }
                q--;
```
if문이랑 else문에 중복되는 코드가 있어서 이렇게 바꿀 수 있을것같아요!
근데 코드가 중복되는게 알고리즘은 이해가 더 잘되는 것 같아요ㅋㅋㅋㅋ